dart/test/environment_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:sentry/sentry.dart';
+import 'package:test/test.dart';
+
+import 'mocks.dart';
+import 'mocks/mock_environment_variables.dart';
+
+void main() {
+  // See https://docs.sentry.io/platforms/dart/configuration/options/
+  // and https://github.com/getsentry/sentry-dart/issues/306
+  group('Environment Variables', () {
```

Comment:
we need to implement `tearDown` here and call Sentry.close() because Sentry is a static class and has state